### PR TITLE
Don't attempt to moc generate files that don't have QOBJECT.

### PR DESCRIPTION
**Pull request by clalancette**
This avoids a warning.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

This removes a warning seen on Linux that looks like:

```
Note: No relevant classes found. No output generated.
```

for each of the two files.

### CODE REVIEW

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14643/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9396/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12307/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14787/)

**Review by ahcorde — CHANGES_REQUESTED** [review]
do we need a tf2 release https://build.ros2.org/job/Rpr__rviz__ubuntu_focal_amd64/123/console ?

**Comment by clalancette** [comment]
> do we need a tf2 release https://build.ros2.org/job/Rpr__rviz__ubuntu_focal_amd64/123/console ?

Yeah, I think we do.  I was holding off because of some other stuff going on there (particularly https://github.com/ros2/geometry2/pull/420).

**Review by wjwwood — APPROVED** [review]
lgtm, I don't think you really need to wait for the Rpr job, but it's up to the maintainers.

**Review by ahcorde — APPROVED** [review]

**Comment by clalancette** [comment]
I'm going to go ahead and merge this, because the failing PR job is not because of this.  Thanks for the reviews!

**Pull request merged**
